src/einsteinpy/symbolic/riemann.py

```diff
@@ -79,9 +79,9 @@ def from_christoffels(cls, chris, parent_metric=None):
             r = (int(i / dims)) % (dims)
             s = (int(i / (dims ** 2))) % (dims)
             t = (int(i / (dims ** 3))) % (dims)
-            temp = sympy.diff(arr[t, s, n], syms[r]) - sympy.diff(arr[t, r, n], syms[s])
+            temp = sympy.diff(arr[t, s, n], syms[r]) - sympy.diff(arr[t, s, r], syms[n])
             for p in range(dims):
-                temp += arr[p, s, n] * arr[t, p, r] - arr[p, r, n] * arr[t, p, s]
+                temp += arr[p, s, n] * arr[t, p, r] - arr[p, s, r] * arr[t, p, n]
             riemann_list[t][s][r][n] = sympy.simplify(temp)
```

Comment:
Changes look good. The math equivalent to this new code is:
![image](https://user-images.githubusercontent.com/31299799/107017005-9f5f3580-67c4-11eb-9f27-1de6073ed82e.png),
which is the correct expression for R. For the old code, we have:
![image](https://user-images.githubusercontent.com/31299799/107017543-4512a480-67c5-11eb-992b-daf79b72ad35.png).

Compare with [Riemann Curvature Tensor on Wikipedia](https://en.wikipedia.org/wiki/Riemann_curvature_tensor#Coordinate_expression) with:
```
rho : t
sigma : s
mu : r
nu : n
```

Reply:
Furthermore, we should include the full expression in the docstring for easy comparison and reference. Something like this ([Line 58](https://github.com/einsteinpy/einsteinpy/blob/a63f2f9ff7e9010d80d7cbd4a519fd8b523ab865/src/einsteinpy/symbolic/riemann.py#L58-L61)):
```
"""
Get Riemann Tensor calculated from Christoffel Symbols. Reimann Tensor is given as:

.. math::

   R^{t}{}_{s r n}=\Gamma^{t}{}_{s n, r} - \Gamma^{t }{}_{s r, n } + \Gamma^{p}{}_{s n}\Gamma^{t}{}_{p r} - \Gamma^{p}{}_{s r}\Gamma^{t}{}_{p n}

Parameters
----------
...
"""
```